docker-compose.yml

```diff
@@ -3,10 +3,10 @@ services:
   p2p-editor:
     build: .
     ports:
-      - "8000:8000"
+      - "3617:3617"
     volumes:
       - ".:/usr/src/app"
   signalhub:
     image: "santaka/signalhub"
     ports:
-      - "8080:8080"
```

Comment:
Do we need to expose to the host machine the Signalhub port? Or it is enough being able to communicate between the app and signalhost inside the containers?

Reply:
Yes, it is a requirement. The browser is the one who communicates with signalhub.

I've increase by 1 the port number (3618) :nerd_face:

Reply:
By the way, what do you think about having a Dockerfile for signalhub within the project, instead of "santaka/signalhub" image. I don't want to maintain that image.

Reply:
Sure thing. Thanks for the quick reply 😊
My bad! 😅